snowplow-android-core/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
+
+    defaultConfig {
+        minSdkVersion 11
+        targetSdkVersion 21
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        debug {
+            buildConfigField "String", "TRACKER_LABEL", "\"andr-$project.version\""
+            buildConfigField "String", "TRACKER_VERSION", "\"$project.version\""
+        }
+        release {
```

Comment:
Hey @duncan, as far as I am aware we only need those vars available so that we can pass them through with events.  So as long as this still makes the information available to the Tracker I don't see an issue.